extensions/amp-carousel/0.2/test-e2e/test-carousel.js

```diff
@@ -111,10 +111,10 @@ describes.endtoend('AMP carousel', {
     await controller.takeScreenshot('screenshots/after-reset.png');
   });
 
-  it('should have the correct scroll position when resizing', async() => {
+  it.skip('should have the correct scroll position when resizing', async() => {
```

Comment:
Does this one need to be skipped? I seem to be able to run it fine locally.

Reply:
You're right, I unskipped it